manifests/init.pp

```diff
@@ -7,6 +7,7 @@
 # @param gpgkey name of the GPG key filename in the repo
 # @param use_release_package enable/disable the puppet-tools-release package installation. When disabled, we will configure the repo as yumrepo resource
 # @param yumrepo_base_url configure the full repo URL, useful when you don't exactly mirror yum.puppet.com
+# @param manage_repo when true, a repo will be added to install bolt. Useful when you manage repos via Satellite. See also $use_release_package
```

Comment:
I would not call out Satellite specifically.

Reply:
removed it